aries-site/src/pages/_app.js

```diff
@@ -59,15 +60,159 @@ const backgroundImages = {
+                  action =
+                    '#' +
+                    sectionArray[0].trim().replace(/\s+/g, '-').toLowerCase();
+                }
+                for (let i = 0; i < Object.keys(sectionArray).length; i++) {
+                  finalSectionlist[i] =
+                    sectionArray[i].charAt(0).toUpperCase() +
+                    sectionArray[i].slice(1).toLowerCase();
+                }
+                let newUpdate;
+                tokenName = `${justName
+                  ?.toLowerCase()
+                  .replace(/\s+/g, '-')}-last-visited`;
+                if (window.localStorage.getItem(tokenName)) {
+                  if (
+                    window.localStorage.getItem(tokenName) >
+                    new Date(data[i].merged_at).getTime()
+                  ) {
+                    //history of them visiting the page before, and that visit was after the newest update was sent
+                    newUpdate = false;
+                  } else {
+                    //history of them visiting the page before, but that visit was before the newest update was sent
+                    newUpdate = true;
+                  }
+                } else {
+                  //have never seen the page before
+                  newUpdate = true;
+                }
+                temporaryHistory[justName] = {
+                  type: typeChange,
+                  description: notificationList[j + 2].slice(1, -1),
+                  date: data[i].merged_at,
+                  sections: sectionArray,
+                  action: action,
+                  update: newUpdate,
+                };
+              }
+            }
+          }
+        }
+        window.localStorage.setItem(
+          'update-history',
+          JSON.stringify(temporaryHistory),
+        );
+        setWholeViewHistory(temporaryHistory);
+        setPageUpdateReady(true);
+      })
+      .then(() => {
+        if (name) {
+          let tokenName = `${name
+            ?.toLowerCase()
+            .replace(/\s+/g, '-')}-last-visited`;
+          let dateNow = new Date().getTime();
+          window.localStorage.setItem(tokenName, dateNow);
+        }
+      })
+      .catch(error => console.error(error));
+  }, []);
+
   useEffect(() => {
     const handleRouteChange = () => {
       const skipLinks = document.querySelector('#skip-links');
       skipLinks.focus();
+
+      if (typeof window !== 'undefined') {
+        let viewHistory = JSON.parse(
+          window.localStorage.getItem('update-history'),
+        );
+        let name = router.asPath;
+        let nameArray = name.split('/');
+        name = nameArray[Object.keys(nameArray).length - 1];
+        name = name.charAt(0).toUpperCase() + name.slice(1);
+
+        let noQueryName = name.split('?')[0];
+        let tokenName = `${noQueryName
+          ?.toLowerCase()
+          .replace(/\s+/g, '-')}-last-visited`;
+        let dateTime = new Date().getTime();
```

Comment:
```suggestion
        const dateTime = new Date().getTime();
```
Does this get modified?